config/views.view.search_content.yml

```diff
@@ -76,14 +76,18 @@ display:
               iasc_space: search_result
             'entity:node':
               action_point: search_result
-              announcement: teaser
-              contact: teaser
+              announcement: search_result
+              article: search_result
+              contact: search_result
               oa_discussion_post: search_result
-              oa_event: teaser
-              oa_wiki_page: teaser
+              oa_event: search_result
+              oa_section: search_result
+              oa_team: search_result
+              oa_wiki_page: search_result
               oa_worktracker_task: search_result
-              panopoly_landing_page: teaser
-              panopoly_page: teaser
+              page: search_result
+              panopoly_landing_page: search_result
+              panopoly_page: search_result
```

Comment:
I think this is a regression. Should be teasers

Reply:
my bad, will fix it